### PR TITLE
ADEN-7483 Applying new changes from app Prebid

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/bidders/prebid/adapters/appnexus.js
Comment (fraszczakszymon) [inline src/bidders/prebid/adapters/appnexus.js]: What about changing context (defined placements) in app repository instead of creating special logic for handling recovery case here?

File: src/bidders/prebid/adapters/index-exchange.js
Comment (fraszczakszymon) [inline src/bidders/prebid/adapters/index-exchange.js]: What about changing context (defined placements) in app repository instead of creating special logic for handling recovery case here?

File: src/bidders/prebid/adapters/appnexus.js
Comment (dmnsobczak) [reply src/bidders/prebid/adapters/appnexus.js]: That's a great idea. I gonna handle this in app repository.

File: src/bidders/prebid/adapters/index-exchange.js
Comment (dmnsobczak) [reply src/bidders/prebid/adapters/index-exchange.js]: That's a great idea. I gonna handle this in app repository.

File: src/bidders/prebid/adapters/appnexus-ast.js
Comment (fraszczakszymon) [inline src/bidders/prebid/adapters/appnexus-ast.js]: Keep in context please

Comment (dmnsobczak) [reply src/bidders/prebid/adapters/appnexus-ast.js]: Done.

File: src/bidders/prebid/adapters/appnexus.js
Comment (fraszczakszymon) [inline src/bidders/prebid/adapters/appnexus.js:10]: Can we remove this comment?

File: src/bidders/a9/index.js
Comment (fraszczakszymon) [inline src/bidders/a9/index.js:69]: `slot-service` already has kind of slot "states". It isn't exposed. But maybe we should is it here? Let's don't block this pull request... but we can add some todo note for later :( 

Your call

File: src/index.js
Comment (fraszczakszymon) [inline src/index.js:13]: It turns out that we don't need to concat bidders to global-bundle. Am I right? https://github.com/Wikia/ad-products/blob/dev/maintenance/build-bundle.sh#L4

File: src/bidders/prebid/prebid-helper.js
Comment (jacekbj) [inline src/bidders/prebid/prebid-helper.js]: `lazyLoadSlots` will always be evaluated to `true`, event if it's empty. I think it's not necessary here.

File: src/bidders/prebid/adapters/appnexus.js
Comment (dmnsobczak) [reply src/bidders/prebid/adapters/appnexus.js:10]: Emm, it is already removed :)

File: src/bidders/a9/index.js
Comment (dmnsobczak) [reply src/bidders/a9/index.js:69]: This PR is already quite big. I think it's better to do this in different commit, separate task (also this change can be quite majorish).

File: src/index.js
Comment (dmnsobczak) [reply src/index.js:13]: I think you right. Removing.

File: src/bidders/prebid/prebid-helper.js
Comment (dmnsobczak) [reply src/bidders/prebid/prebid-helper.js]: You are right, it can be removed.

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**
